src/main/java/gregtech/loaders/recipe/MachineRecipeLoader.java

```diff
@@ -54,15 +54,15 @@ private MachineRecipeLoader() {
 
     public static void init() {
```

Comment:
Registration is quite inconsistent and hard to follow. Like where whole init function should be registered and where just part of the file.